app/Http/Controllers/UKCP/Token.php

```diff
@@ -35,47 +34,31 @@ public function create()
             return Redirect::route('mship.manage.dashboard')
                 ->withError('An unknown error occured, please contact Web Services.');
         }
-
-        $latestId = $this->ukcp->getValidTokensFor(auth()->user())->first()->id;
-        $tokenPath = 'ukcp/tokens/' . auth()->user()->id . '/' . $latestId . '.json';
-        Storage::disk('local')->put($tokenPath, $newToken);
-
-        return $this->viewMake('ukcp.token.create')->with('newToken', $latestId);
+        return redirect()->route('ukcp.guide')->withSuccess('Tokens Updated!');
```

Comment:
Do you really want to go back to the guide?

Reply:
I think this is best. Provides a clear way to download the new key (the big download button), and for new people they don't need to click the link to view what they need to do (= hopefully less questions!)

Reply:
And it already did go back to the guide (ukcp.token.create is the view that is now renamed ukcp.token.guide, and the ukcp.guide route makes that view)